apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -519,7 +519,10 @@ export default function Tutorial({ useCase }: Props) {
        */
       const timeout: ReturnType<typeof setTimeout> = setTimeout(() => {
         if (history.location.pathname !== "/")
-          handleStartPostOnboarding(connectedDevice.modelId, true, () => history.push("/"));
+          handleStartPostOnboarding({
```

Comment:
- this function's signature changed, now passing options in an object for more clarity
- also removing the "true" which was to a mock post onboarding (this shouldn't be the mock post onboarding but the real one, if implemented